packages/ember-testing/lib/helpers.js

```diff
@@ -55,6 +55,7 @@ asyncHelper('visit', visit);
 
   @method click
   @param {String} selector jQuery selector for finding element on the DOM
+  @param {Object} context A DOM Element, Document, or jQuery to use as context
```

Comment:
@ladyleet yeah given the choices, which you quoted <http://api.jquery.com/jQuery/#jQuery1>, in the description `Object` seems fine,